skins/oasis/css/core/ads.scss

```diff
@@ -120,7 +120,7 @@ $video-interactive-area-z-index: $vpaid-z-index + 1;
 
 .wikia-ad {
 	&.slot-animation {
-		@include transition(max-height, 0.8s, ease, 0);
+		@include transition(max-height, 1.6s, ease, 0);
```

Comment:
Fix imports in lines 14-15, please. I know the lines hasn't been added with this work but since we're touching Porvata and our `ads.scss` - we can fix them :) More details in "Use sass imports in App with caution" e-mail from @latata from Mar 24 2017 :blush: